### ISSUE TITLE
Show 'Installed' when certain textures are already installed?

### ISSUE THREAD

**Issue by Vezqi** [AlphaConsole/AlphaConsoleElectron#3]
I'm talking about what's in this image. Maybe, when it's installed, grey it out, and and have it show 'Installed', or even have it say something like install, where it will actually remove the files from the user's machine, and prompt asking if they really want to uninstall it? I clicked this a few times wondering why it wouldn't download, until I figured it was already installed. 
![image](https://user-images.githubusercontent.com/27045730/40750837-7dca2cde-641d-11e8-9705-9c4d2c88c6d6.png)

**Comment by jd0920** [comment]
I'm glad you pointed that out. So I was working on a solution and at this point I've managed to get a text to show up on installed packages. I am working on implementing a change for the download button tomorrow. 

Preview: https://imgur.com/a/41KjnwY

**Comment by jd0920** [comment]
I've changed it from "Installed" to just a check mark as a placeholder for some kind of downloaded icon.
![screenshot from 2018-05-31 01-33-36](https://user-images.githubusercontent.com/38387163/40771662-c172ece6-6472-11e8-9a9d-6a0db0ec9fef.png)

**Comment by Vezqi** [comment]
Looks a bit difficult to read.. I was thinking something more like this, with the button having a hex color of `#323232`.
![image](https://user-images.githubusercontent.com/27045730/40793910-8fcffbee-64b2-11e8-80ea-2e063db23f20.png)

**Comment by burnss9** [comment]
I think the check mark is fine. Also a toggle to filter out installed ones would be nice

**Comment by Vezqi** [comment]
I agree the check mark is fine too, but making it more visible at least would be perfect.

**Comment by jd0920** [comment]
@Vezqi Definitely, I tried making it as visible as I can haha. I was considering grabbing a icon off a website but then the credit would need to be put somewhere so I just put an html checkmark as a placeholder but I might be able to play around with colours... Maybe add a background colour? I don't know design can still be discussed. We are a little limited due to the size of the squares and whatnot so I think it should be an icon.

**Comment by jd0920** [comment]
@burnss9 The filter is definitely doable. Each installed package has the `installed` class.
